dashboard/app/views/admin_reports/retention_stages.html.haml

```diff
@@ -0,0 +1,57 @@
+- content_for :head do
+  %script{src:'https://www.google.com/jsapi'}
+  %script{src:'https://www.gstatic.com/charts/loader.js'}
+
+%h1
+  = "Retention for Stages #{@stages.join(', ')}"
+
+%br
+
+#chart-stage-level-counts
+#table-stage-level-counts
+
+%br
+
+
+- content_for :body_scripts do
+  :javascript
+    stages = #{@stages};
+    stats = #{@retention_stats};
+    // Load the Visualization API and the appropriate packages, setting a
+    // callback to run when the API is loaded.
+    google.load('visualization', '1.0', {'packages':['corechart', 'table']});
+    google.setOnLoadCallback(drawChartsAndTables);
+
+    // The callback that creates and populates the data tables, instantiates the
+    // charts and tables, and draws them.
+    function drawChartsAndTables() {
+      drawChartAndTable(
+        'stage-level-counts', 'Stage Level Counts', 'Stages', stages, stats);
+    }
```

Comment:
Note the awkwardness of having the drawChartsAndTables method be a pass-through to drawChartAndTable is intentional. My intent is to soon consolidate similar JS across on other pages.